admin/pkg/authtoken/authtoken.go

```diff
@@ -131,6 +131,14 @@ func (t *Token) SecretHash() []byte {
 	return hashed[:]
 }
 
+// Prefix returns a safe, partial display string for the token, e.g., "rill_usr_abcdefghij"
+func (t *Token) Prefix() string {
```

Comment:
The thing is this needs to work even if `t.Secret` is empty (as long as `t.ID` is set). Can you:
1. Note that in the docstring.
2. Add a test case that loops e.g. 100 times and checks that the prefix is correct for random tokens where the `Secret` is empty?